app/lib/rack_exceptions_app.rb

```diff
@@ -0,0 +1,18 @@
+class RackExceptionsApp
+  def self.call(env)
+    if env['REQUEST_PATH'] =~ /^\/api\/.*$/
+      [503, { 'Content-Type' => 'text/plain' }, ['Service Unavailable']]
+    else
+      @body ||= action_view.render(template: 'errors/service_down', layout: 'layouts/error')
```

Comment:
:+1:! Prob not for this PR, but it would be good to backport this template to the other `service_unavailable` page so they always share conten?

Reply:
This is now an internal server error page, rather than a service unavailable page.